Remove line length restriction on ruff CI checks

We seem to be ignoring the line length restriction the ruff CI checks throw up, so best to turn them off. They'll confused new contributors and could lead to us ignoring more important ruff errors because they get hidden in the noise and we get used to ignoring them.

Example: https://github.com/ACCESS-NRI/spack-packages/pull/72/files

Error:

```
Ruff (E501)
packages/mom5/package.py:22:89: E501 Line too long (191 > 88 characters)
```

--- a/.ruff.toml
+++ b/.ruff.toml
@@ -1,6 +1,5 @@
 select = ["E","F"]
-ignore = ["E221","E241","E272","E731", "F403","F405","F821"]
-line-length = 88
+ignore = ["E221","E241","E272","E501","E731", "F403","F405","F821"]
 output-format = "github"
 
 # "E129", "F999" aren't supported

--- a/packages/access-om2-bgc/package.py
+++ b/packages/access-om2-bgc/package.py
@@ -1,0 +1,30 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# Copyright 2023 ACCESS-NRI
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+class AccessOm2Bgc(BundlePackage):
+    """ACCESS-OM2 bundle contains the coupled CICE5 and MOM5 (BGC variant) models."""
+
+    homepage = "https://www.access-nri.org.au"
+
+    git = "https://github.com/ACCESS-NRI/ACCESS-OM2-BGC.git"
+
+    maintainers = ["harshula"]
+
+    version("latest")
+
+    variant("deterministic", default=False, description="Deterministic build.")
+
+    depends_on("libaccessom2+deterministic", when="+deterministic")
+    depends_on("libaccessom2~deterministic", when="~deterministic")
+    depends_on("cice5+deterministic", when="+deterministic")
+    depends_on("cice5~deterministic", when="~deterministic")
+    depends_on("mom5+deterministic type=ACCESS-OM-BGC", when="+deterministic")
+    depends_on("mom5~deterministic type=ACCESS-OM-BGC", when="~deterministic")
+
+    # There is no need for install() since there is no code.

--- a/packages/access-om2-bgc/package.py
+++ b/packages/access-om2-bgc/package.py
@@ -8,7 +8,7 @@
 from spack.package import *
 
 class AccessOm2Bgc(BundlePackage):
-    """ACCESS-OM2 bundle contains the coupled CICE5 and MOM5 (BGC variant) models."""
+    """ACCESS-OM2-BGC bundle contains the coupled CICE5 and MOM5 (BGC variant) models."""
 
     homepage = "https://www.access-nri.org.au"
 

